### ISSUE TITLE
Validate commit messages

### ISSUE THREAD

**Issue by awinogradov** [qfox/violence#2]
angular: https://github.com/Famous/famous-angular/blob/master/validateCommitMsg.js

**Comment by qfox** [comment]
Yeah, looks great. I'll take it for plugin to check commit messages!

**Comment by awinogradov** [comment]
:+1:

**Comment by Guria** [comment]
project dead completely?

**Comment by qfox** [comment]
Wow, it was a half of a year.

Nope, just waiting for jscs3.0 for some reason.

Do you want to contribute some good parts?
